data/categories.txt

```diff
@@ -287,7 +287,7 @@ cs:3Banka|peníze
 sk:3Banka|peniaze
 nl:3Bank|geld
 zh-Hant:1銀行|錢
-pl:3Bank|pieniądze
+pl:3Placówka banku|bank|pieniądze
```

Comment:
As you wish. I think about "Bank" as the name of institution not a place and so I changed it according to this.